doc/README/README.driver

```diff
@@ -46,6 +46,12 @@ ready for user interaction after the image file was saved.
 N.B.: If you are adroit, you could use the virtual X11 server program Xvfb
 to create images as indicated above with no need for an actual X11 display.
 A sample script is given later -- search forward to find the string 'Xvfb'.
+Also see the script @snapshot_volreg for a practical example.
```

Comment:
This PR seems to include other changes, not related to CMAKEification... makes it harder to review